pkg/graph/resolvers/system_intake_grb_review_form.go

```diff
@@ -0,0 +1,69 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/jmoiron/sqlx"
+
+	"github.com/cms-enterprise/easi-app/pkg/models"
+	"github.com/cms-enterprise/easi-app/pkg/sqlutils"
+	"github.com/cms-enterprise/easi-app/pkg/storage"
+)
+
+// UpdateSystemIntakeGRBReviewType updates the GRB review type on a system intake
+func UpdateSystemIntakeGRBReviewType(
+	ctx context.Context,
+	store *storage.Store,
+	input models.UpdateSystemIntakeGRBReviewTypeInput,
+) (*models.UpdateSystemIntakePayload, error) {
+	return storage.UpdateSystemIntakeGRBReviewType(
+		ctx,
+		store,
+		input.SystemIntakeID,
+		input.GrbReviewType,
+	)
+}
+
+func UpdateSystemIntakeGRBReviewForm(
+	ctx context.Context,
+	store *storage.Store,
+	input models.UpdateSystemIntakeGRBReviewFormInput,
+) (*models.UpdateSystemIntakePayload, error) {
+	return sqlutils.WithTransactionRet[*models.UpdateSystemIntakePayload](ctx, store, func(tx *sqlx.Tx) (*models.UpdateSystemIntakePayload, error) {
+		// Fetch intake by ID
+		intake, err := store.FetchSystemIntakeByIDNP(ctx, tx, input.SystemIntakeID)
+		if err != nil {
+			return nil, err
+		}
+
+		if input.GrbReviewType != nil {
```

Comment:
This is the other section that would be affected if you switch to omittable. In MINT, apply changes handled this for us by also returning the struct as a MAP[string]interface. Omittable allows us to have similar behavior.